### PR TITLE
Actionbar

### PR DESCRIPTION
**Pull request by lgirault**

### CODE REVIEW

**Comment by lgirault** [comment]
Here is an example of use : 


        <Actionbar {...this.props}>

          <Button buttonClassNames={['mcs-actionbar-button-add']} text="+" />

          <Button text="X" secondaryBar="superActionBar" />

          <Button text="Y" secondaryBar="anotherBar" />


          <SecondaryActionbar id="superActionBar">
            This is a superb secondary action bar !
          </SecondaryActionbar>
          <SecondaryActionbar id="anotherBar">
            Another awesome bar !
          </SecondaryActionbar>
        </Actionbar>

File: app/react/src/containers/Actionbar/ActionbarButton.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+    const clickAction = evt => {
+      setSecondaryActionBar(secondaryBar);
+      if (this.props.onClick) { this.props.onClick(evt); }
+    };
+
+    return (
+      <div className={classNames(containerClassNames)}>
+        <button className={classNames(buttonClassNames)} onClick={clickAction} {...this.props}>
```
Comment (pvign) [inline app/react/src/containers/Actionbar/ActionbarButton.js]: use `this.props.other` i/o `this.props`

File: app/react/src/containers/Actionbar/ActionbarButton.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    const selected = currentSecondaryBar !== null && currentSecondaryBar === secondaryBar;
+
+    const selectionClasses = ['selectionArrow'];
+
+    if (!selected) { selectionClasses.push('hidden'); }
+
+
+    const clickAction = evt => {
+      setSecondaryActionBar(secondaryBar);
+      if (this.props.onClick) { this.props.onClick(evt); }
```
Comment (pvign) [inline app/react/src/containers/Actionbar/ActionbarButton.js]: `onClick` should be declared [here](https://github.com/MEDIARITHMICS/mediarithmics-navigator/pull/381/files#diff-052449477119c854c9fa0c06409cec68R19)

File: app/react/src/containers/Actionbar/Actionbar.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+class Actionbar extends Component {
+
+  constructor(props) {
+    super(props);
+    this.renderSecondary = this.renderSecondary.bind(this);
+  }
+
+
+  isSecondaryBarContent = elt =>
```
Comment (pvign) [inline app/react/src/containers/Actionbar/Actionbar.js]: use `isSecondaryBarContent() {}` syntax

File: app/react/src/containers/Actionbar/Editbar.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      onClickLeftButton,
+      onClickCloseButton,
+      leftButtonText
+    } = this.props;
+
+    return (
+      <div className="mcs-editbar" >
+        <BreadcrumbBar className="mcs-editbar-breadcrumb" />
+        <div className="mcs-editbar-buttons">
+          <button className="mcs-editbar-save-button" onClick={onClickLeftButton}> {leftButtonText} </button>
```
Comment (pvign) [inline app/react/src/containers/Actionbar/Editbar.js]: Use `<FormattedMessage id={leftButtonText} />`

File: app/react/src/containers/Actionbar/Editbar.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+
+Editbar.propTypes = {
+  leftButtonText: PropTypes.string,
+  onClickLeftButton: PropTypes.func,
+  onClickCloseButton: PropTypes.func
+};
+
+Editbar.defaultProps = {
+  leftButtonText: 'Save',
```
Comment (pvign) [inline app/react/src/containers/Actionbar/Editbar.js]: Use an i18n key i/o hard coded value

File: app/react/src/containers/Actionbar/Editbar.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  onClickLeftButton: () => {},
+  onClickCloseButton: () => {}
+};
+const mapStateToProps = state => ({
+  activeWorkspace: state.sessionState.activeWorkspace,
```
Comment (pvign) [inline app/react/src/containers/Actionbar/Editbar.js]: `activeWorkspace` isn't used

File: app/react/src/containers/Actionbar/BreadcrumbBar.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+    const buildItem = elt => {
+      const item = elt.url ? <Link to={elt.url}>{elt.name}</Link> : elt.name;
+      return <Breadcrumb.Item>{item}</Breadcrumb.Item>;
+    };
+    const sep = <svg width="7" height="12" viewBox="0 0 7 12" xmlns="http://www.w3.org/2000/svg"><title>EC88C228-76F7-4171-933A-063605048718</title><path d="M6.748 6.594l-5.28 5.16a.874.874 0 0 1-1.216 0 .827.827 0 0 1 0-1.189L4.924 6 .252 1.435a.827.827 0 0 1 0-1.189.874.874 0 0 1 1.216 0l5.28 5.16a.828.828 0 0 1 0 1.188z" fill="#AEAEAE" /></svg>;
+    return (
+      <Breadcrumb separator={sep} {...this.props}>
+        {path.map(buildItem)}
+        <Breadcrumb.Item>Plop</Breadcrumb.Item>
```
Comment (pvign) [inline app/react/src/containers/Actionbar/BreadcrumbBar.js]: Plop ?

Comment (lgirault) [inline app/react/src/containers/Actionbar/BreadcrumbBar.js]: >_< left prinln

File: app/react/src/containers/Actionbar/ActionbarButton.js [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+ActionbarButton.propTypes = {
+  text: PropTypes.string.isRequired,
+  imageUrl: PropTypes.string,
+  secondaryBar: PropTypes.string,
+  containerClassNames: PropTypes.arrayOf(PropTypes.string),
+  buttonClassNames: PropTypes.arrayOf(PropTypes.string),
+  setSecondaryActionBar: PropTypes.func, // eslint-disable-line react/require-default-props
+  currentSecondaryBar: PropTypes.func, // eslint-disable-line react/require-default-props
```
Comment (pvign) [inline app/react/src/containers/Actionbar/ActionbarButton.js]: `currentSecondaryBar` should be  a `PropTypes.string`

File: app/react/src/containers/Actionbar/BreadcrumbBar.js [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+class BreadcrumbBar extends Component {
+
+  render() {
+    const {
+      path
+    } = this.props;
+
+    const buildItem = elt => {
+      const item = elt.url ? <Link to={elt.url}>{elt.name}</Link> : elt.name;
+      return <Breadcrumb.Item>{item}</Breadcrumb.Item>;
```
Comment (pvign) [inline app/react/src/containers/Actionbar/BreadcrumbBar.js]: Missing a  `key` property on `<Breadcrumb.Item>`

**Pull request merged**